test/integration-test.py

```diff
@@ -202,6 +200,36 @@ def run_node_test():
     wait_for_ocsp_revoked(cert_file_pem, "../test-ca.pem", ee_ocsp_url)
     return 0
 
+def run_caa_node_test():
+    s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+    try:
+        s.connect(('localhost', 4000))
+    except socket.error, e:
+        print("Cannot connect to WFE")
+        die(ExitStatus.Error)
+
+    if subprocess.Popen('npm install', shell=True).wait() != 0:
```

Comment:
This and the runNode command below could use some factoring out.

Is there a reason to have the Node CAA test separate from the main Node test rather than an additional step? That would allow the npm install step to be shared.